grpc/oob/bmc/bmc.go

```diff
@@ -359,6 +368,64 @@ func (m Action) BMCReset(ctx context.Context, rType string) (err error) {
 	return nil
 }
 
+// DeactivateSOL functionality for machines.
+func (m Action) DeactivateSOL(ctx context.Context) (err error) {
+	tracer := otel.Tracer("pbnj")
+	ctx, span := tracer.Start(ctx, "client.DeactivateSOL")
+	defer span.End()
+
+	host, user, password, parseErr := m.ParseAuth(m.DeactivateSOLRequest.Authn)
+	if parseErr != nil {
+		return parseErr
+	}
+	span.SetAttributes(attribute.String("bmc.host", host), attribute.String("bmc.username", user))
+	m.SendStatusMessage("working on SOL session deactivation")
+
+	opts := []bmclib.Option{
+		bmclib.WithLogger(m.Log),
+		bmclib.WithPerProviderTimeout(common.BMCTimeoutFromCtx(ctx)),
+		bmclib.WithIpmitoolPort("623"),
+	}
+
+	client := bmclib.NewClient(host, user, password, opts...)
+
+	err = client.Open(ctx)
+	if err != nil {
```

Comment:
to keep this in a tighter scope so that it can't accidentally be re-used later, you could single line it:
```go
if err := client.Open(ctx); err != nil {
// etc
```